client/apps/settings/client/curations/venice_admin.coffee

```diff
@@ -97,6 +102,28 @@ module.exports = VeniceAdmin = React.createClass
         textarea {
           className: 'bordered-input'
           placeholder: 'Description'
+          name: 'description'
           defaultValue: @state.curation.get 'description'
-          onChange: @onInputChange
+          onChange: @onDescriptionChange
+        }
+      div {className: 'field-group sub-articles'},
+        label {}, 'Related Articles'
+        AutocompleteList {
+          url: "#{sd.API_URL}/articles?published=true&q=%QUERY"
+          placeholder: "Search articles by title..."
+          filter: (articles) ->
+            for article in articles.results
+              { id: article.id, value: "#{article.title}, #{article.author?.name}"}
+          selected: (e, item, items) =>
+            subArticles = @state.curation.get 'sub_articles'
+            subArticles = _.pluck items, 'id'
+            @onChange 'sub_articles', subArticles
+          removed: (e, item, items) =>
+            subArticles = @state.curation.get 'sub_articles'
```

Comment:
What's the purpose of having this `subArticles` if it gets overridden on the next line?